semperfiwebdesign/all-in-one-seo-pack
# Improve help text for new tooltips

Issue #2435

## Proposed changes

This PR significantly improves the look of our help text for title format fields.

I've also made two minor changes to two other help text strings, namely for the "Create RSS Sitemap" setting and "Rewrite Titles" setting, and removed the %category% macro in the help text for the Post Title Format field since it's deprecated.

## Types of changes

What types of changes does your code introduce?

- Improves existing functionalty
- Improves existing code

## Checklist


## Testing instructions

You should look at all tooltips for the Title Format fields and make sure they look good.
You should also look at the tooltips for the "Create RSS Sitemap" and "Rewrite Titles" setting.

## Further comments

For the macros within the title format field tooltips, we now use HTML description lists - https://www.w3schools.com/tags/tag_dt.asp (courtesy of @EkoJR).

I'd also like to improve all help text in the General Settings menu by removing all the "Check this option to" and "This option does" parts but that's something to look at later.
